tests/X86/DATAXFER/MOVBE.S

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2019 Trail of Bits, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+TEST_BEGIN_64(MOVBE_GPRv_MEMv_16, 1)
+TEST_INPUTS(
+    0)
```

Comment:
would recommend putting other inputs that should distinguish working from not working, then do
`push ARG1_64` or something like that (look at other tests), and tehn `movbe ..., [rsp]`